client/go/outline/electron/vpnlinux/routing_linux.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2024 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpnlinux
+
+import (
+	"errors"
+	"log/slog"
+
+	perrs "github.com/Jigsaw-Code/outline-apps/client/go/outline/platerrors"
+	"github.com/vishvananda/netlink"
+)
+
+type RoutingRule struct {
+	table int
+	rule  *netlink.Rule
```

Comment:
These may also be replaced by NetworkManager: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/7/html/networking_guide/configuring-policy-based-routing-to-define-alternative-routes#routing-traffic-from-a-specific-subnet-to-a-different-default-gateway

Reply:
Done